### ISSUE TITLE
Excluding js (false) renders "true"

### ISSUE THREAD

**Issue by theredcoder** [kbparagua/paloma#79]
Digging through Paloma this afternoon, and noticed when I exclude js(false) for a controller or specific action, it renders "true" within the insert_paloma_hook div tag.  Running Paloma's test_app (http://localhost:3000/main/prevent) renders "true" too.  Is this intended?

Thanks ahead of time!

**Comment by kbparagua** [comment]
@theredcoder what version is this? Thanks.

**Comment by theredcoder** [comment]
@kbparagua the app is using:
```
paloma (4.2.0)
jquery-rails (4.0.3)
rails (4.2.1)
ruby (2.1.2)
```

**Comment by rowanoulton** [comment]
+1 have run into this today. Not certain, but it seems to occur when `js` is defined more than once in the same action. For example:

```ruby
def create
  js :new # we want to run the js from the #new action here too
  
  # .... does something ...
  
  js false # decided that we no longer require js, eg. after successful save of an object
end
```

**Comment by kbparagua** [comment]
Fixed as of `4.2.1`

**Comment by rowanoulton** [comment]
Awesome, thank you!
